src/components/SimpleHeader/styles.js

```diff
@@ -23,18 +23,18 @@ export const styles = StyleSheet.create({
   leftContainer: {
     marginRight: 'auto',
     paddingLeft: 10,
-    paddingBottom: 8,
+    paddingBottom: 12,
   },
   titleContainer: {
     position: 'absolute',
     width: '100%',
     flexDirection: 'row',
     justifyContent: 'center',
-    paddingBottom: 8,
+    paddingBottom: 12,
   },
   rightContainer: {
     marginLeft: 'auto',
     paddingRight: 10,
-    paddingBottom: 8,
+    paddingBottom: 12,
```

Comment:
Just a reminder, be sure that button is clickable both ways (especially left container/ back button). A quick way to test it is using Dev Menu > Inspector.